wp-cli/vip-migrations.php

```diff
@@ -20,9 +20,10 @@ function cleanup( $args, $assoc_args ) {
 		global $wpdb;
 
 		$dry_run = Utils\get_flag_value( $assoc_args, 'dry-run' );
+		$yes = Utils\get_flag_value( $assoc_args, 'yes' );
```

Comment:
Minor: Can we call it `skip-confirm` or something more intuitive? (I know `--yes` is a pattern that wp-cli uses, but it's not a great flag).